nupic/embodied/envs/real_robots/policies.py

```diff
@@ -20,8 +20,11 @@
 #
 # ------------------------------------------------------------------------------
 import numpy as np
-from real_robots.policy import BasePolicy
 
```

Comment:
Wouldn't that break import the class altogether, since it inherits from BasePolicy?